channeldb/invoices.go

```diff
@@ -631,8 +631,12 @@ func (d *DB) QueryInvoices(q InvoiceQuery) (InvoiceSlice, error) {
 //
 // When the preimage for the invoice is unknown (hold invoice), the invoice is
 // marked as accepted.
+//
+// TODO: Store invoice cltv as separate field in database so that it doesn't
+// need to be decoded from the payment request.
```

Comment:
I've also noticed that at times the pubkey recovery from the signature can show up in profiles when benchmarking payments. We may also want to add a flag to the decode routing that lets it skip obtaining the public key as well.

Reply:
Ok, but out of scope for this PR? Also the fix direction could either be to create a light payment decoding function or move the final cltv value in its own db key.

Reply:
Oh yeah it's totally out of the scope, just a drive by comment.